DTS-STN/canadian-dental-care-plan
# AB#3303 -- Updated the French and English translations for the Terms & Conditions page

### Description
Updated the French and English translations for the Terms & Conditions page

### Related Azure Boards Work Items
[AB#3303](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3303)

### Screenshots (if applicable)
n/a

### Checklist

### Test Instructions
View T&C apply page to confirm it conforms with the May release final figma design

### Additional Notes
n/a
